plugins/rapture/nexus-rapture-baseapp/src/main/baseapp/packages/rapture-theme/sass/src/Ext/form/field/Base.scss

```diff
@@ -8,10 +8,25 @@
  * cls: 'x-form-readonly'
  */
 
-.x-form-field[readonly].x-form-field:not([role="combobox"]) {
+@mixin nx-form-field-readonly {
   color: $color-dark-gray;
-  background-color: $color-smoke;
```

Comment:
Why did you remove the background color from read-only fields? Seems like a regression.